File: javascript/rna-transcription/rna-transcription.js
const CODE_RNA = {
    A: 'U',
    G: 'C',
    C: 'G',
    T: 'A'
  }

export const toRna = (str) => {
  return str
    .split('')
    .map(i => CODE_RNA[i])
    .join('');
}

File: ruby/rna-transcription/rna_transcription.rb
# RNA Transcription
class Complement
  CODE = { 'G' => 'C', 'C' => 'G', 'T' => 'A', 'A' => 'U' }.freeze
  def self.of_dna(dna)
    dna.split('').map { |i| CODE[i] }.join
  end
end
